migrations/versions/5ec5c84ba61e_.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
+
+Revision ID: 5ec5c84ba61e
+Revises: d1eae8b9893e
+Create Date: 2017-10-17 18:21:00.174015
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils as su
+import sqlalchemy_searchable as ss
+
+
+# revision identifiers, used by Alembic.
+revision = '5ec5c84ba61e'
+down_revision = 'd1eae8b9893e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    op.add_column('queries', sa.Column('search_vector', su.TSVectorType(),
+                                       nullable=True))
+
+    ss.sync_trigger(conn, 'queries', 'search_vector',
+                    ['name', 'description', 'query'])
+
+
+def downgrade():
+    conn = op.get_bind()
+
+    ss.drop_trigger(conn, 'queries', 'search_vector')
+    op.drop_index('ix_queries_search_vector', table_name='queries')
```

Comment:
This is a byproduct of `ss.sync_trigger`?

Reply:
Yep, although weirdly it raised a ProgrammingError exception when I tried roll it back, will look into this.

Reply:
This was an oversight when setting up SA-Searchable. The migration that I fixed in https://github.com/getredash/redash/pull/2041/commits/eb6966fa83d2b44978bf5b49db0df052d11c8170 now reflects adding an index for the search vector column. Another call to `db.configure_mappers()` was needed to have the mapper processor of SA-Searchable to correctly add the index.

Reply:
👍 

> Another call to db.configure_mappers() was needed

It seems that it's at the top level of `redash.models`. So I guess the one in `redash.cli.database` is redundant?

Reply:
True indeed, good catch, removed it in https://github.com/getredash/redash/pull/2041/commits/36aeef776d610a266d1c9dfa166be10bb5d1726a.